src/main/webapp/index-script.js

```diff
@@ -274,8 +274,13 @@ function generateResult(place) {
     place.name,
     place.international_phone_number,
     place.website,
-    place.vicinity + " away"
+    place.vicinity
   ];
+
+  tidbits = tidbits.filter(function (element) {
+    return element != null;
+  });
+
```

Comment:
will this have an effect on the payout of the elements on the page since you're essentially skipping elements?

Reply:
This filter only skips attributes that are null. It skips elements that will be displayed as ```undefined```